From 24858efb8924405be6b61c5eb3a88635055308b3 Mon Sep 17 00:00:00 2001
From: ViKing <viking.yfy@qq.com>
Date: Fri, 28 Aug 2026 19:37:13 +0800
Subject: [PATCH] net: dsa: qca8k: mark QCA four-byte tag test build

Update the temporary RE-CS-08 marker for the first-stage QSDK qca_4b
transport test.

Signed-off-by: ViKing <viking.yfy@qq.com>
---
 drivers/net/dsa/qca/qca8k-8xxx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/dsa/qca/qca8k-8xxx.c b/drivers/net/dsa/qca/qca8k-8xxx.c
index 22b41a1..92e5ef1 100644
--- a/drivers/net/dsa/qca/qca8k-8xxx.c
+++ b/drivers/net/dsa/qca/qca8k-8xxx.c
@@ -2380,7 +2380,7 @@ qca8k_sw_probe(struct mdio_device *mdiodev)
 	priv->info = of_device_get_match_data(priv->dev);
 	if (qca8k_is_qca8386(priv))
 		dev_info_once(priv->dev,
-			      "RE-CS-08 firmware debug tag: 20260828-155644-native-qca-sfp-los\n");
+			      "RE-CS-08 firmware debug tag: 20260828-193235-qca4b-stage1\n");
 
 	priv->reset_gpio = devm_gpiod_get_optional(priv->dev, "reset",
 						   GPIOD_OUT_HIGH);
-- 
2.53.0

